ext/pdo_oci/tests/pdo_oci_attr_action.phpt

```diff
@@ -0,0 +1,52 @@
+--TEST--
+PDO_OCI: Attribute: Setting session action
+--SKIPIF--
+<?php
+if (!extension_loaded('pdo') || !extension_loaded('pdo_oci')) die('skip not loaded');
+require(dirname(__FILE__).'/../../pdo/tests/pdo_test.inc');
+PDOTest::skip();
+?>
+--FILE--
+<?php
+
+require(dirname(__FILE__) . '/../../pdo/tests/pdo_test.inc');
+
+$query = 'select action from v$session where sid = (select distinct sid from v$mystat)';
```

Comment:
It can be cleaner to use `sid = sys_context('USERENV', 'SID')`.

Reply:
👍 Changed on these tests.